pipelines/toast_satellite_sim.py

```diff
@@ -553,6 +663,9 @@ def main():
             nse = tt.OpSimNoise(out="noise", realization=mc)
             nse.exec(data)
 
+            # add sky signal
+            add_sky_signal(args, comm, data, totalname="noise", signalname=signalname)
```

Comment:
do you prefer I rename the key to "signal" now?

Reply:
Sure, that would make sense.

Reply:
ok, I will rename it to `tot_signal` because `signal` is already the sky signal